### ISSUE TITLE
Unit with space

### ISSUE THREAD

**Issue by VelizarVESSELINOV** [kinverarity1/lasio#363]
Units like '1000 lbf', '1000 psi' are split as Unit '1000' and Value 'lbf'

Example:
```
HKLA            .1000 lbf                                  :(RT)
```

It will be nice to correctly catch the units.

Proposal check the Energistics unit catalog: https://www.energistics.org/energistics-unit-of-measure-standard/

or eventual fix:

```Python
def is_float(value):
    """Is float value."""
    try:
        float(value)
        return True
    except ValueError:
        return False

# for each curve c:
if is_float(c.unit) and len(c.value) > 1:
     c.unit = c.unit + ' ' + c.value
     c.value = ''  # or = None (don't know what it is your empty value default)

```

**Comment by kinverarity1** [comment]
Sounds like a good idea, thanks! If anyone submits this as a PR however, be aware we would want to only join the first whitespace-delimited part of ``c.value`` to the unit.

**Comment by kinverarity1** [comment]
@VelizarVESSELINOV This has been fixed in a limited way by #390 in that units consisting of a numeric value like "1000" followed by a space and the actual unit, will be parsed as a single unit, e.g. "1000 lbf".
